ably-ios/ARTAuth.h

```diff
@@ -30,7 +30,7 @@ ART_ASSUME_NONNULL_BEGIN
 @property (nonatomic, weak) ARTLog *logger;
 
 @property (art_nullable, readonly, getter=getClientId) NSString *clientId;
-@property (art_nullable, nonatomic, readonly, strong) ARTAuthTokenDetails *tokenDetails;
+@property (art_nullable, nonatomic, readwrite, strong) ARTAuthTokenDetails *tokenDetails;
```

Comment:
Why is this `readwrite`?  I assume this means a consumer of this API could explicitly update the `tokenDetails` without using the correct API.  Is this intentional?  If so, why?